src/main/java/handler/InputHandler.java

```diff
@@ -0,0 +1,90 @@
+package handler;
+
+import ui.UI;
+import exception.DukeException;
+
+public class InputHandler {
+
+    private static final String TODO = "todo";
+    private static final String LIST = "list";
+    private static final String DEADLINE = "deadline";
+    private static final String EVENT = "event";
+    private static final String BYE = "bye";
+    private static final String MARK = "mark";
+    private static final String UNMARK = "unmark";
+    private static final String DELETE = "delete";
+
+    public static void handleInput() {
+        boolean exit = false;
```

Comment:
Consider renaming the boolean to `isExit`, in adherence to the naming convention.